### PR TITLE
WIP more tests

### PR DESCRIPTION
**Pull request by oskarrough**

### CODE REVIEW

File: cypress/integration/find.js [hunk 1]
```diff
@@ -40,5 +40,23 @@ describe('Find', function() {
 		assertQuery('&gh internet4000/radio4000', 'https://github.com/internet4000/radio4000')
 		assertQuery('&gh internet4000/radio4000', 'https://github.com/internet4000/radio4000')
 	})
+
+	it('has a method to replace a placeholder', () => {
```
Comment (4www) [inline cypress/integration/find.js:36]: not sure how this test helps?

Comment (oskarrough) [reply cypress/integration/find.js:36]: i added it as placeholder, needs real tests but wan't sure how the method works

Comment (4www) [reply cypress/integration/find.js:36]: alright!

**Review by 4www — APPROVED** [review]
cool stuff, nice direction.
If we can keep the code-base small and well tested, it is going to be real fun!
Thanks for the addition :}

**Pull request merged**